docs/sphinx/install.rst

```diff
@@ -428,9 +425,145 @@ when connecting via SSH, which will launch the CUDA Quantum container and connec
+you can use the NGC CLI to launch and interact with workloads in DGX Cloud.
+The following sections detail how to do that, and how to connect JupyterLab and/or VS Code
+to a running CUDA Quantum job in DGX Cloud.
+
+.. _dgx-cloud-setup:
+
+Get Started
++++++++++++++++++++++++++++++++
+
+To get started with DGX Cloud, you can 
+`request access here <https://www.nvidia.com/en-us/data-center/dgx-cloud/trial/>`__.
+Once you have access, `sign in <https://ngc.nvidia.com/signin>`__ to your account,
+and `generate an API key <https://ngc.nvidia.com/setup/api-key>`__. 
+`Install the NGC CLI <https://ngc.nvidia.com/setup/installers/cli>`__
+and configure it with 
+
+.. code-block:: console
+
+    ngc config set
+
+entering the API key you just generated when prompted.
+
+.. note::
+
+  The rest of this section assumes you have CLI version 3.33.0. If you 
+  have an older version installed, you can upgrade to the latest version using the command
+
+  .. code-block:: console
+
+      ngc version upgrade 3.33.0
+  
+  See also the `NGC CLI documentation <https://docs.ngc.nvidia.com/cli/index.html>`__
+  for more information about available commands.
+
+You can see all information about available compute resources and ace instances
+with the command 
+
+.. code-block:: console
+
+    ngc base-command ace list
+
+Confirm that you can submit a job with the command
+
+.. code-block:: console
+
+    ngc base-command job run \
+      --name Job-001 --total-runtime 60s \
+      --image nvcr.io/nvidia/nightly/cuda-quantum:latest --result /results \
+      --ace <ace_name> --instance <instance_name> \
+      --commandline 'echo "Hello from DGX Cloud!"'
+
+replacing `<ace_name>` and `<instance_name>` with the name of the ace and instance you want 
+to execute the job on.
+You should now see that job listed when you run the command
+
+.. code-block:: console
+
+    ngc base-command job list
+
+Once it has completed you can download the job results using the command
+
+.. code-block:: console
+
+    ngc base-command result download <job_id>
+
+replacing `<job_id>` with the id of the job you just submitted.
+You should see a new folder named `<job_id>` with the job log that contains 
+the output "Hello from DGX Cloud!".
+
+For more information about how to use the NGC CLI to interact with DGX Cloud, 
+we refer to the `NGC CLI documentation <https://docs.ngc.nvidia.com/cli/index.html>`__.
+
+Use JupyterLab
++++++++++++++++++++++++++++++++
+
+Once you can :ref:`run jobs on DGX Cloud <dgx-cloud-setup>`, you can launch an interactive job 
+to use CUDA Quantum with `JupyterLab <https://jupyterlab.readthedocs.io/en/latest/>`__ 
+running on DGX Cloud:
+
+.. code-block:: console
+
+    ngc base-command job run \
+      --name Job-interactive-001 --total-runtime 600s \
+      --image nvcr.io/nvidia/nightly/cuda-quantum:latest --result /results \
+      --ace <ace_name> --instance <instance_name> \
+      --port 8888 --commandline 'jupyter-lab-setup <my-custom-token> --port=8888'
```

Comment:
I can't test this exactly as written, but it seems like the second `--port=8888` in this line would cause problems because the `jupyter-lab-setup` script is using `$@` for the token.

Reply:
I did test this locally (by building and pushing a new image with these changes to the local registry), and this works fine. The $@ will just act as a way to "append" the rest of the command line right after token=

Reply:
That's a bit surprising to me since the `$@` is in quotes (i.e. `token="$@"`), but I'll be the first to admit that I wasn't sure without testing it, so thanks for testing it.